uniplot/layer_factory.py

```diff
@@ -78,50 +78,67 @@ def render_points(xs: List[NDArray], ys: List[NDArray], options: Options) -> NDA
     if not options.force_ascii:
         scaling_factor_width = 2
         scaling_factor_height = 4 if options.character_set == "braille" else 2
-    matrix: NDArray = np.array([])
+
+    # ideally we 1 create empty on
+
+    height, width = scaling_factor_width * options.height, scaling_factor_height * options.width
+    matrix: NDArray = np.zeros((height, width), dtype=int)
 
```

Comment:
I inizialize a empty matrix and over write on it, faster are more concise, I pass the layer for the colors